build.proj

```diff
@@ -20,7 +20,11 @@
           DependsOnTargets="MSBuildWorkaroundTarget;
                             RestoreDotnetCliBuildFramework">
 
-    <Exec Command="$(DotnetStage0) publish -o $(DotnetCliBuildDirectory)/bin --framework netcoreapp1.0 /p:GeneratingPropsFile=true" 
+    <PropertyGroup>
+      <GeneratingPropsFileArg Condition=" '$(GeneratingPropsFile)' == 'true' ">/p:GeneratingPropsFile=true</GeneratingPropsFileArg>
```

Comment:
Yeah I think so. I can change that. But do you mind me changing that separately? It will require changes to our run-build.ps1 and .sh scripts too since it's a rename of an existing property.